packages/eslint-plugin/tests/rules/consistent-type-assertions.test.ts

```diff
@@ -144,6 +145,12 @@ ruleTester.run('consistent-type-assertions', rule, {
         },
       ],
     },
+    {
+      code: "console.log('Hello, world!');",
+      languageOptions: {
+        parser: espree,
```

Comment:
Instead of bringing in espree and updating it manually etc.
Could we just do a similar thing to what ESLint does for TS parsing -- parse it with the parser, dump the json blob out and then define a fake parser module eg
```
languageOptions: {
  parser: {
    parse: () => { type: 'Program', ... },
  },
},
```

WDYT?

Reply:
+1 to this suggestion

Reply:
I did this, then realized we call `parser.parse` manually in another file. I did that with a separate string literal.